tests/Hedgehog.Benchmarks/Hedgehog.Benchmarks.fsproj

```diff
@@ -2,20 +2,23 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Could that be the reason why those tests fail on your local?

Reply:
I don't know what you mean.  Can you elaborate?

This specific line is in the Benchmark project.  It is changed only to match the same change in the main test project.  That change was part of a workaround I found that gets the tests to _pass_.  Before I made that change, two tests failed.  To repeat, two tests fail for me on the first commit and all tests pass for me on the second commit.

Reply:
Nah, you're right. I didn't notice it was on the benchmarks project. 👍